unyt/_unit_lookup_table.py

```diff
@@ -544,7 +553,29 @@
         ("hp", ("horsepower",)),
         ("oz", ("ounce",)),
         ("cal", ("calorie",)),
-        ("BTU", ("british_thermal_unit",)),
+        (
+            "BTU",
+            (
+                "british_thermal_unit",
+                "btu",
+            ),
+        ),
+        (
+            "MMBTU",
+            (
+                "million_british_thermal_unit",
```

Comment:
I don't think we should support this alias as I doubt anyone would ever try to type it. Happy to be persuaded otherwise if you disagree.

Reply:
I'm not attached to it. I only added it for symmetry with BTU.